llvm/lib/TargetParser/Host.cpp

```diff
@@ -228,6 +228,7 @@ StringRef sys::detail::getHostCPUNameForARM(StringRef ProcCpuinfoContent) {
         .Case("0xd14", "cortex-r82ae")
         .Case("0xd02", "cortex-a34")
         .Case("0xd04", "cortex-a35")
+        .Case("0xd8f", "cortex-a320")
```

Comment:
There is also `llvm/unittests/TargetParser/Host.cpp` which reverse tests these, but it doesn't appear to be very well maintained (and I'm not sure how useful it is; perhaps that's why it's not been updated recently, and because there are so many cores now supported, it needs removing or some TLC)

Reply:
Added the test regardless.